src/lib/tradingView/utils.ts

```diff
@@ -37,17 +38,17 @@ const getOhlcValues = ({
   orderbookOpen?: number;
   orderbookClose?: number;
 }) => {
-  const useOrderbookCandles =
+  const showOrderbookCandles =
     orderbookCandlesToggleOn &&
-    trades <= MIN_NUM_TRADES_FOR_ORDERBOOK_PRICES &&
+    trades <= MAX_NUM_TRADES_FOR_ORDERBOOK_PRICES &&
```

Comment:
Are we opposed to displaying candles for a single trade? `<` instead of `<=`?

Reply:
i am confirming with matt and adam 🤔

Reply:
@jaredvu this is fine